package.json

```diff
@@ -8,7 +8,8 @@
     "clean": "rm -rf dist",
     "lint": "eslint --ext .js,.jsx,.mjs,.ts,.tsx .",
     "start": "webpack --config webpack-config.js --watch",
-    "test": "jest"
+    "test": "jest",
+    "test:snapshot": "jest --updateSnapshot"
```

Comment:
This can also be achieved by running `npm run test -- -u`

Reply:
I feel there is still benefit to having an explicit npm script to do this, however we can document this approach when documentation is created. I've added a TODO item on the TODO list in issue #17